src/app/lib/color-demo/color-demo.component.html

```diff
@@ -0,0 +1,118 @@
+        <hc-swatch-demo-component hex="#553e36" name="brown"></hc-swatch-demo-component>
+        <hc-swatch-demo-component hex="#776c7f" name="purple-gray"></hc-swatch-demo-component>
+    </div>
+    <div class="palette-row">
+        <hc-swatch-demo-component hex="#f5ed56" name="yellow"></hc-swatch-demo-component>
+        <hc-swatch-demo-component hex="#e7c447" name="yellow-orange"></hc-swatch-demo-component>
+        <hc-swatch-demo-component hex="#ebba82" name="tan"></hc-swatch-demo-component>
+        <hc-swatch-demo-component hex="" name=""></hc-swatch-demo-component>
+        <hc-swatch-demo-component hex="" name=""></hc-swatch-demo-component>
+        <hc-swatch-demo-component hex="" name=""></hc-swatch-demo-component>
+    </div>
+
+    <hr>
+
+    <h4>General Color Variables</h4>
+    <p class="description">
+        Some general-use color variables, referencing variables above, to decouple color name from an intended purpose.
+    </p>
+
+    <ul class="color-var-list">
+        <li><span class="mini-swatch" [style.backgroundColor]="'#00aeff'"></span><strong>$primary-brand</strong> - $blue</li>
+        <li><span class="mini-swatch" [style.backgroundColor]="'#00a859'"></span><strong>$primary-action</strong> - $green</li>
+        <li><span class="mini-swatch" [style.backgroundColor]="'#6e53a3'"></span><strong>$primary-alt-action</strong> - $purple</li>
+        <li><span class="mini-swatch" [style.backgroundColor]="'#f13c45'"></span><strong>$destructive-action</strong> - $red</li>
+        <li><span class="mini-swatch" [style.backgroundColor]="'#a1a1a1'"></span><strong>$neutral-action</strong> - $gray-400</li>
+        <li><span class="mini-swatch" [style.backgroundColor]="'#cc2027'"></span><strong>$error</strong> - $deep-red</li>
+        <li><span class="mini-swatch" [style.backgroundColor]="'#333333'"></span><strong>$text</strong> - $offblack</li>
+    </ul>
+
+    <hr>
+
+    <h4>Stratifying and Adding Colors</h4>
+    <p class="description">
+        There may be times that a product will need to expand the color palette, or use a variation on
+        exisiting colors. In these cases we recommend one of two approaches:
+    </p>
+
+    <ol>
+        <li>If importing the sass variables into your application, first try manipulating an existing color variable using the built in sass functions
+            <code>darken()</code> or <code>lighten()</code>, or the library provided <code>shade()</code> or <code>tint()</code> functions.
+            <pre>
+            <code>
+            &#x00040;import &quot;../colors&quot;;
+    
+            .my-component &#123;
+                background-color: darken($slate-gray-100, 5%);
+                color: tint($dark-blue, 5%);
+            &#125;
+            </code>
+            </pre>
+        
+        
+        </li>
+        <li>If that doesn't suffice, or you are not using the sass variables, try using <a hc-button href="coolors.co" target="_blank">coolors.co</a>
```

Comment:
This is really minor @corykon - put make sure and add http:// to your link to coolors.co so it doesn't try and link to our cashmere site.

Reply:
good catch! thank you